release/salt-yum-defs/terraform.tfvars

```diff
@@ -3,17 +3,15 @@ repos = [
     repo_prefix = "repo/saltstack/salt/linux/"
     yum_prefix  = "yum.defs/saltstack/salt/"
     salt_versions = [
-      "3004",
-      "3003.3",
-      "3003",
+      "3004.2",
     ]
   },
   {
     repo_prefix = "repo/archive/saltstack/salt/linux/"
     yum_prefix  = "yum.defs/saltstack/salt/"
     salt_versions = [
-      "2019.2.8",
-      "2019.2.5",
+      "3004",
+      "3003.3",
```

Comment:
This is related to the previous comment.  These versions are not in Salt's archive location from what I can tell.

Reply:
They are under the `py3` path...

* https://s3.archive.repo.saltproject.io/py3/redhat/7/x86_64/archive/3003.3/
* https://s3.archive.repo.saltproject.io/py3/redhat/7/x86_64/archive/3004/

The `yum` path is only for python2 packages, and these are not published for python2...

Reply:
salt-reposync pulls _both_ the python2 and python3 packages for each version specified...

https://github.com/plus3it/salt-reposync/blob/08cc9f020395339fe87e023cc74a32e30801d0e3/modules/repo/main.tf#L28


https://github.com/plus3it/salt-reposync/blob/08cc9f020395339fe87e023cc74a32e30801d0e3/modules/repo/main.tf#L36